app/controllers/orderitems_controller.rb

```diff
@@ -0,0 +1,48 @@
+class OrderitemsController < ApplicationController
+
+  #this method hooks up to button in cart to delete an orderitem, OR you might end up redirected here if you update at orderitem quantity to zero.
```

Comment:
I'm loving the comments but some of them are really long. Break them up into multiple lines if needed.